src/popup/modules/InitQrCode.js

```diff
@@ -92,6 +91,7 @@ export const initiationProcess = Promise.all([qrCreatorInit, userInterfaceInit])
  * @returns {Promise}
  */
 function getTabWithValidUrl() {
+    const queryBrowserTabs = () => browser.tabs.query({active: true, currentWindow: true});
```

Comment:
Okay, but actually you do not need to save it in any variable...
You only call it once in line 106.